site/en/r2/tutorials/load_data/images.ipynb

```diff
@@ -143,7 +143,7 @@
       "source": [
         "### Retrieve the images\n",
         "\n",
-        "Before you start any training, you'll need a set of images to teach the network about the new classes you want to recognize. We've created an archive of creative-commons licensed flower photos to use initially."
+        "Before you start any training, you will need a set of images to teach the network about the new classes you want to recognize. We have already created an archive of creative-commons licensed flower photos to use initially:"
```

Comment:
Thanks!
While you're in here :)
Can you remove usage of "we" or replace with "you" —here and below
Google style is to prefer second person: https://developers.google.com/style/highlights

Reply:
Done + other minor changes as per the Google style guide. Thanks for the help @lamberta